prod/westeurope/notifications/sandbox/resource_group/terragrunt.hcl

```diff
@@ -0,0 +1,12 @@
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_resource_group?ref=v3.0.3"
```

Comment:
delete this resource group

Reply:
why? don't we need a dedicated resource group for the storage account?

Reply:
no, maybe a previous misunderstanding. sorry